modules/osdk-control-compat.adoc

```diff
@@ -76,13 +77,8 @@ com.redhat.openshift.versions: "v4.6-v4.8" <1>
 +
```

Comment:
IMO: a nit here
your Operator's bundle image (`bundle.Dockerfile`)

bundle.Dockerfile is the name of the ` Operator's bundle image` when we scaffold with SDK but that could have any name.

c/c @adellape

Reply:
Updated the lead-in sentence:

> ... in your Operator's bundle image. For example, if your project was generated using the Operator SDK, update the `bundle.Dockerfile` file:

Reply:
That seems ok for .. you can close this one. :-)